sdk/python/kfp/compiler/_container_builder.py

```diff
@@ -20,20 +20,23 @@
 
 SERVICEACCOUNT_NAMESPACE = '/var/run/secrets/kubernetes.io/serviceaccount/namespace'
 GCS_STAGING_BLOB_DEFAULT_PREFIX = 'kfp_container_build_staging'
+GCR_DEFAULT_IMAGE_SUFFIX = 'kfp_container'
 
 class ContainerBuilder(object):
   """
   ContainerBuilder helps build a container image
   """
-  def __init__(self, gcs_staging=None, namespace=None):
+  def __init__(self, gcs_staging=None, gcr_image_tag=None, namespace=None):
```

Comment:
why default to gcr? should this be platform agnostic?

Reply:
It's supposed to be platform agnostic at best. The goal for this PR is to configure the GCR path. Once https://github.com/kubeflow/pipelines/issues/1700 is fixed, the container builder will be platform agnostic.

Reply:
What's ETA for #1700?

Reply:
The estimated efforts for #1700 is about 3 days, give or take. 
In terms of ETA, it is not P0 for now. We welcome community contribution on supporting multiple other platforms such as AWS, Azure etc.